p4-16/psa/PSA.mdk

```diff
@@ -451,6 +451,8 @@ enum HashAlgorithm {
   crc32_custom,
   crc16,
   crc16_custom,
+  ones_complement16,  /// One's complement 16-bit sum used for IPv4 headers,
+                      /// TCP, and UDP.
   random,           /// are random hash algorithms useful?
   identity
```

Comment:
can you also please move identity to be the first item on the list of algorithms (per @vgurevich's request)?